train.py

```diff
@@ -617,7 +617,10 @@ def update_from_result(name, test, result):
         i_time_avg = (i*i_time_avg + i_time)/(i+1)
         i_left = iterations/test_interval - (i+1)
         time_left = i_time_avg * i_left
-        time_str = time.strftime('%H:%M:%S', time.gmtime(time_left))
+        time_str = time.strftime('%j:%H:%M:%S', time.gmtime(time_left))
```

Comment:
This is a bit hacky and fragile.  Instead:
```
time_str = str(datetime.timedelta(seconds = time_left))
```

Reply:
I applied the changes, thanks for letting me know about the datetime package! My first method was indeed hacky